### PR TITLE
changed salutation

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
When you are over-quota, in the over-quota warning email you receive, the user's full name if available will be used in the email salutation rather than the user's username.

### CODE REVIEW

**Comment by hyi** [comment]
@sblack-usu Can you take a quick look of this very small PR and see if you can approve it?

**Comment by hydrobuild** [comment]
Build finished.

File: hs_core/tasks.py [hunk 1]
```diff
@@ -206,8 +206,16 @@ def manage_task_nightly():
                 uq.remaining_grace_period = 0
             uq.save()
 
-            uemail = u.email
-            msg_str = 'Dear ' + u.username + ':\n\n'
+            if u.first_name and u.last_name:
```
Comment (sblack-usu) [inline hs_core/tasks.py]: How about this?
```
msg = 'Dear {}:\n\n'.format('{} {}'.format(u.first_name or '', u.last_name or '').trim())
```

**Comment by hyi** [comment]
@sblack-usu Thanks for reviewing it and for the suggestion for the one liner. However, that one liner does not include the case that if both first name and last name were empty, the username would be used. I personally think the ifs are logically clearer that the condensed one liner for this specific case. I have just merged this.

**Comment by aphelionz** [comment]
@hyi Was there an issue for this?

**Comment by hyi** [comment]
@aphelionz Yes, here is the issue #2606 created by @dtarb this PR addressed.

**Comment by aphelionz** [comment]
Thank you @hyi

**Pull request merged**